### ISSUE TITLE
Bogus behaviour of MarkOccurrenceAction

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-eclipse#252]
There is a bogus behaviour of MarkOccurrenceAction(Contributor)

you can reproduce it with these steps

- create a mydsl xtext dsl
- create a mydsl1 xtext dsl (or use xtend)
- start a runtime eclipse
- open a mydsl and mydsl1/xtend file side by side.
- restart the runtime eclipse

=> you get the toggle mark occurrences button twice in the toolbar.

**Comment by cdietrich** [comment]
![issue252](https://cloud.githubusercontent.com/assets/238736/25617467/ca50955e-2f42-11e7-9b19-2bb3b2e70b45.png)

**Comment by cdietrich** [comment]
possible solution

```
ActionContributionItem item = new ActionContributionItem(getAction());
				item.setVisible(false);
				toolBarManager.add(item);
```

**Comment by cdietrich** [comment]
Fixed in Xtext 2.12
